ckan/requirements.txt

```diff
@@ -83,24 +86,25 @@ requests==2.25.0
 rfc3987==1.3.8
 Routes==1.13
 rq==1.0
-# pin setuptools until we get python buildpack 1.7.58
```

Comment:
do we still need these? if we move the pin/comment to the requirements.in it shouldn't get blown away.

Reply:
nevermind, its already there.